src/effects/VST/VSTEffect.h

```diff
@@ -97,7 +97,7 @@ struct VSTEffectSettings
    std::vector<char> mChunk;
 
    // Fallback data used when the chunk is not available.
-   std::map<wxString, std::optional<std::pair<int,double> > > mParamsMap;
+   std::unordered_map<wxString, std::optional<double> > mParamsMap;
```

Comment:
You change this back to `unordered_map` because the parallel iteration of two maps is no longer required so the iteration order of elements can be left unpredictable.

Change the line `#include <map>` to fix macOS compilation.

Reply:
ok